<?xml version="1.0"?>
<!-- 
  Wrapper launch file for the lebai specific io_service node.
-->
<launch>
  <!-- IP of robot (or PC running simulation) -->
  <arg name="robot_ip" doc="IP of controller" />
  <param name="robot_ip_address" type="str" value="$(arg robot_ip)" />
  <!-- load the correct version of the robot state node -->
  <node name="io_service" pkg="lebai_driver" type="io_service_node.py"  output="screen">
  </node>    
</launch>